src/air/forms.py

```diff
@@ -250,56 +250,60 @@ class CheeseModel(BaseModel):
 
         class CheeseForm(air.AirForm):
             model = CheeseModel
+
+    Used with FastAPI's dependency injection system:
+        class CheeseModel(pydantic.BaseModel):
+            name: str
+            age: int
+
+        class CheeseForm(air.AirForm):
+            model = CheeseModel
+
+        @app.post("/cheese")
+        async def cheese_form(cheese: Annotated[CheeseForm, Depends(CheeseForm())]):
+            if cheese.is_valid:
+                return air.Html(air.H1(cheese.data.name))
+            return air.Html(air.H1(air.Raw(str(len(cheese.errors)))))
     """
-    if json_schema_extra is None:
-        json_schema_extra = {}
+    schema_extra: dict[str, Any] = dict(json_schema_extra or {})
```

Comment:
I switched it back to `json_schema_extra` and the changes `ruff` made to accomodate the extra length was unpleasant. Nice fix for handling formatting issues.